zebra/zebra_dplane.c

```diff
@@ -139,7 +139,7 @@ struct dplane_intf_info {
 #define DPLANE_INTF_CONNECTED   (1 << 0) /* Connected peer, p2p */
 #define DPLANE_INTF_SECONDARY   (1 << 1)
 #define DPLANE_INTF_BROADCAST   (1 << 2)
-#define DPLANE_INTF_HAS_DEST    (1 << 3)
+#define DPLANE_INTF_HAS_DEST    DPLANE_INTF_CONNECTED
```

Comment:
This is probably not a good solution, but with the change `DPLANE_INTF_HAS_DEST` implies `DPLANE_INTF_CONNECTED` and vice versa. Should this flag be removed?